lib/player_setup.rb

```diff
@@ -0,0 +1,16 @@
+module TicTacToe
+  class PlayerSetup 
+    attr_reader :p1
+    attr_reader :p2
+
+    def player_assignment(marker1, marker2, console_ui, game_type)
+      if game_type.human_v_human
+        @p1 = Human.new(marker1, console_ui)
+        @p2 = Human.new(marker2, console_ui)
+      elsif game_type.human_v_simp_comp
+        @p1 = Human.new(marker1, console_ui)
```

Comment:
This is a good example of a class that is initializing the same class in multiple places. What do you think about extracting the HumanPlayer initialization into a private method that can be invoked in the 3 places you are initializing a Human object? 
```
def human_player(marker1, console_ui)
  Human.new(marker1, console_ui)
end
```